doc/cephadm/monitoring.rst

```diff
@@ -14,8 +15,27 @@ performance.  Ceph users have three options:
 #. Skip the monitoring stack completely.  Some Ceph dashboard graphs will
    not be available.
 
-Deploying monitoring with cephadm
----------------------------------
+The monitoring stack consists of `Prometheus <https://prometheus.io/>`_,
+Prometheus exporters (:ref:`mgr-prometheus`, `Node exporter
+<https://prometheus.io/docs/guides/node-exporter/>`_), `Prometheus Alert
+Manager <https://prometheus.io/docs/alerting/alertmanager/>`_ and `Grafana
+<https://grafana.com/>`_.
+
+.. note::
+
+  Please note that Prometheus' security model presumes that untrusted users
+  have access to the Prometheus HTTP endpoint and logs. They have access to
+  all time series information contained in the database, plus a variety of
+  operational/debugging information.
+
+  However, Prometheus' HTTP API is limited to read-only operations. No
+  configurations can be changed using the API and secrets are not exposed.
```

Comment:
```suggestion
  However, Prometheus' HTTP API is limited to read-only operations. Configurations
  can *not* be changed using the API, and secrets can not be exposed.

Reply:
I've partly adapted the sentence to the one you provided, though I'd like to keep the "are not exposed" part, as particular configuration secrets are not exposed but I'm not so sure about manually added configuration options.

Reply:
Works for me!